src/vue/backend/src/templates/page.vue

```diff
@@ -20,19 +23,72 @@ export default {
 </script>
 <style lang="scss">
 .content-page {
-    padding: 50px;
-    margin: 83px 50px 0px 255px;
+    // Spacing
+    --space-4: 16px;
```

Comment:
Are you using all of these variables? If this is meant to be global, it would be better in a global style sheet instead of hidden in the page component